appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/model/transition/Operation.kt

```diff
@@ -3,7 +3,11 @@ package com.bumble.appyx.interactions.core.model.transition
 import com.bumble.appyx.interactions.Parcelable
 import com.bumble.appyx.interactions.Parcelize
 
-interface Operation<ModelState> : (ModelState) -> StateTransition<ModelState>, Parcelable {
+interface ModelStateToStateTransition<ModelState> {
+    fun invoke(state: ModelState): StateTransition<ModelState>
+}
```

Comment:
Do we need a separate interface for this? We could just merge it into `Operation`? (+ also, would `operator fun invoke` work?)

Reply:
Good catch. I'll add `invoke` method` to `Operation` interface. And seems like `operator fun invoke` works fine.